docs/home/howto/LOGGING.md

```diff
@@ -46,3 +49,28 @@ specification:
     repo: /var/lib/elasticsearch-snapshots
     logs: /var/log/elasticsearch
 ```
+
+### How to export Elasticsearch data to csv format
+Since v0.8 Epiphany provide posibility to export data from Elasticsearch to CSV using Logstash *(logstash-oss v7.8.1*) along with *logstash-input-elasticsearch (v4.6.2)* and *logstash-output-csv (v3.0.8)* plugin.
+
+To install Logstash in your cluster add **logstash** to feature mapping for *logging, opendistro_for_elasticsearch* or *elasticsearch* group.
+
+`Note: Logstash is disabled by default after installation.`
```

Comment:
Can you explain here why it is disabled from the start? :)

Reply:
Yes I can :)